test/cookbooks/acme_test/recipes/acme_dns_client.rb

```diff
@@ -0,0 +1,110 @@
+node.default['osl-acme']['pebble']['always_valid'] = false
+node.default['osl-acme']['pebble']['command'] = '/usr/local/bin/pebble -config /opt/pebble/test/config/pebble-config.json -dnsserver :8053'
+
+build_essential 'osl-acme-test' do
+  compile_time true
+end
+
+chef_gem 'bcrypt' do
+  compile_time true
+end
```

Comment:
Where is this being used?

Reply:
I'm using it [here](https://github.com/osuosl-cookbooks/osl-acme/pull/17/files#diff-d7a45c50a60c1a3fe0b4b6cdb65835078a8eaed41c66cc75e7ae60090731f4ac) and [here](https://github.com/osuosl-cookbooks/osl-acme/pull/17/files#diff-0322745e4ad572bf464535011935b09f891c0f934d6461b521bf3dc9b9843191R70) (tests only) to generate the password hashes dynamically and store them in the database. We could alternatively store the password hashes in the databag, but I figured it was better to keep the databag as similar to production as possible (and would make updating the tests more difficult).

Reply:
If you made a sql dump like I suggested earlier, would this no longer be needed?